website/content/en/docs/faq.md

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
I'm seeing a lot of AWS Specific FAQs in here. Can we scope this to AWS?

Reply:
In the mean time, any time I find an FAQ question that is duplicated in the concepts section, I'll add a link to it from here. That should help reduce the amount of drift between the two docs.